charts/csi-baremetal-operator/crds/csi-baremetal.dell.com_availablecapacities.yaml

```diff
@@ -1,10 +1,9 @@
-
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.5.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
any reason to upgrade the version here ?

Reply:
The immutablity validation rule, i.e. the block of cr update in crd validation, is not supported in crd generation of controller-gen v0.5.0. To support the block of storagegroup.spec update in crd validation, I have to upgrade controller-gen version to v0.9.2, the earliest version to support the immutablity validation rule in crd generation.